### PR TITLE
Objects Automated Test - LPS-141513 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -2889,11 +2889,71 @@ definition {
...
+
+		AssertElementPresent(
+			key_fieldColumnNumber = "1",
+			key_fieldLabel = "Custom Field B",
+			locator1 = "ObjectAdmin#FIELD_COLUMN_POSITION_ON_ENTRY_PAGE");
+
+		AssertElementPresent(
+			key_fieldColumnNumber = "2",
+			key_fieldLabel = "Custom Field C",
+			locator1 = "ObjectAdmin#FIELD_COLUMN_POSITION_ON_ENTRY_PAGE");
```
Comment (PauloVAS) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:2955]: Please, create a macro for that. Thanks!

**Review by PauloVAS — CHANGES_REQUESTED** [review]
@evanilson-santana,

Please, check the comment and make the necessary changes, thanks.

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase (hunk 1 above)
Comment (evanilson-santana) [reply modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase]: Fixed

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**